news/2025/2025-11-22-osu-world-cup-2025-round-of-16-recap.md

```diff
@@ -0,0 +1,112 @@
+
+### ::{ flag=IT }:: Italy vs. ::{ flag=TR }:: Türkiye
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2619201548&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+It was left to the lone losers' bracket match of Saturday to produce a competitive match, and the battle between Italy and Türkiye delivered in spades. Italy were out to prove to the world that their 0–15 map score in the Group Stage was a fluke, while Türkiye sought to continue a strong run of form that saw them finish 3–0 in Group F.
+
+The match was a roller coaster from the get-go, with Türkiye taking an early break point for a 2–0 lead, only to see Italy find a break point of their own on the highly volatile [NM5](https://osu.ppy.sh/beatmapsets/2462132#osu/5386813) before tying the match up on their own [HD2](https://osu.ppy.sh/beatmapsets/2089293#osu/4378380) pick. Italy continued building momentum by winning another break point on [DT3](https://osu.ppy.sh/beatmapsets/2462142#osu/5386835), but now it was Türkiye's chance to turn the tables. Winning [NM3](https://osu.ppy.sh/beatmapsets/2462129#osu/5386798) at the last moment by a bare margin of 3,320 points, Türkiye opted into [FM3](https://osu.ppy.sh/beatmapsets/2429306#osu/5290020) with their final pick, winning it to earn a single match point at 4–3. Down to their last pick, Italy selected [NM4](https://osu.ppy.sh/beatmapsets/2298768#osu/5167719), but ::{ flag=TR }:: [Raikouhou](https://osu.ppy.sh/users/8007528) led the way with an FC as Türkiye slammed the door shut by a margin of over 800,000 points to win the map and with it the match by a final score of 5–3.
+
+---
+
+Sunday brought forth the majority of the week's matches, getting started early and featuring action throughout the day to the tune of nearly 10 hours of gameplay. While the day started off inauspiciously with Taiwan sweeping the Netherlands, things would heat up over the course of the morning. New Zealand challenged 7th-seeded United Kingdom, with the UK eventually coming out on top 5–3; Japan and Thailand — more on them later — engaged in a fierce battle leading to the week's only tiebreaker.
+
+Several other losers bracket matchups took place with victors eventually including Indonesia, Austria, Sweden, Finland, and Spain. Sunday's afternoon window featured three top 8 seeds getting their tournament under way, as the United States took down France, Germany defeated Brazil, and the Russian Federation became the lone top 8 seed to lose their round of 16 match, with #9 seed China taking them out in a closely fought 5–3 match.
```

Comment:
this is just more summary. should be before both matches, unless you want "saturday" and "sunday" headings